examples/Cavity/CMakeLists.txt

```diff
@@ -1,17 +1,8 @@
-# Cavity
+# ABL
```

Comment:
ABL -> Cavity, it appears all CMakeLists.txt have "ABL" as their comment

Reply:
Solved